lib/builtin.pm

```diff
@@ -162,6 +163,36 @@ numerical argument.
 Returns the largest integer value less than or equal to the given numerical
 argument.
 
+=head2 trim
+
+ $stripped = trim($string);
```

Comment:
For consistency with the rest of the file, please use 4-space indent here

Reply:
Done, but new code should standardize on a 1 space indent if consistency is so desirable.  That gives 3 extra spaces before a long verbatim line will spill over to the next one on any given terminal.  This file doesn't currently have any that come near any likely window width, but new additions could.  And then, does one allow an exception for those wide lines,  living  with an inconsistent indent, does one expend the effort to make the whole file use a shorter indent, or does one say tough about the decreased legibility of our documentation when lines wrap?  The only way to minimize the likelihood of one of those outcomes is to use the shortest feasible indent always..

Our pod has plenty of inconsistent indents, and I have never been bothered when I see them.  Nor have I heard any user complaint.  Verbatim line wrapping, is a problem that decreases the usability of our documentation.  Some formatters, like html, don't pay attention to how many spaces the indent is, the line is trimmed and indented according to style sheet rules.